plugins/apps/threatbus_cif3/plugin.py

```diff
@@ -0,0 +1,98 @@
+from queue import Queue
+import threading
+from cifsdk.client.http import HTTP as Client
+
+from threatbus_cif3.message_mapping import map_to_cif
+import threatbus
+
+"""Threatbus - Open Source Threat Intelligence Platform - plugin for CIFv3"""
+
+plugin_name = "cif3"
+cif = None
+lock = threading.Lock()
+
+
+def validate_config(config):
+    assert config, "config must not be None"
+    config["tags"].get(list)
+    config["tlp"].get(str)
+    config["confidence"].as_number()
+    config["group"].get(str)
+    if config["api"].get(dict):
+        config["api"]["host"].get(str)
+        config["api"]["ssl"].get(bool)
+        config["api"]["token"].get(str)
+
+
+def receive_intel_from_backbone(watched_queue, config):
```

Comment:
```suggestion
def receive_intel_from_backbone(watched_queue, cif, config):
```

Pass the configured cif instance to this function.